sensor/common/sensor/sensor.go

```diff
@@ -167,6 +163,15 @@ func (s *Sensor) Start() {
 		customRoutes = append(customRoutes, koCacheRoute)
 	}
 
+	// Enable endpoint to retrieve vulnerability definitions if local image scanning is enabled.
+	if features.LocalImageScanning.Enabled() {
```

Comment:
might have no need for this new route if `env.LocalImageScanningEnabled.BooleanSetting() == false` as well since this Sensor will *not* be talking to a "local" scanner, but rather the main scanner

Reply:
Good call, in fact, do we still need to check the feature? Wondering if disabling the feature would automatically disable the environment env. It's not clear to me the relationship between them at the moment.